openvpn.tf

```diff
@@ -30,16 +30,16 @@ module "openvpn" {
   freeipa_admin_pw        = var.freeipa_admin_pw
   freeipa_realm           = upper(var.cool_domain)
   hostname                = "vpn.${var.cool_domain}"
-  private_networks        = var.private_networks
-  private_reverse_zone_id = var.private_reverse_zone_id
-  private_zone_id         = var.private_zone_id
+  private_networks        = [data.terraform_remote_state.networking.outputs.vpc.cidr_block]
+  private_reverse_zone_id = data.terraform_remote_state.networking.outputs.public_subnet_private_reverse_zones["10.128.9.0/24"].id
```

Comment:
Can `"10.128.9.0/24"` be replaced with a variable here and below?

Reply:
See commit 5996486.

Reply:
Gravy! 👍